packages/rocketchat-livechat/app/client/lib/_livechat.js

```diff
@@ -2,6 +2,7 @@ this.Livechat = new (class Livechat {
 	constructor() {
 		this._online = new ReactiveVar(null);
 
+		this._language = new ReactiveVar();
```

Comment:
@sampaiodiego Since this is set asynchronously by the new livechat API function, it is not set when we first call GetInitialData. How can I best solve this issue?

Reply:
Unless you explicitly advise otherwise, I'm gonna fetch the translations with a new meteor function that will be called whenever the `Livechat.language` changes.